README.md

```diff
@@ -1,2 +1,156 @@
-# camel-dap-client-sublime
-Extension to leverage debug adpater for Camel in Sublime
+# Sublime Debug Adapter client for Apache Camel
+
+![A breakpoint hit on a Camel route endpoint](./images/sublime-dap.gif)
+
+## Prerequisites
+ - Camel version 3.18+
+ - camel-debug is available on the classpath
+ - have JMX enabled
+
+# How to configure Sublime
+
+In a `~/.config/sublime-text/Packages/Debugger/modules/adapters/camel.py` file:
+```py
+from __future__ import annotations
+from ..typecheck import *
+
+from .import util
+from .. import dap
+from .. import core
+
+import sublime
+import re
+
+class Camel(dap.AdapterConfiguration):
+
+    type = 'camel'
+    docs = 'https://github.com/camel-tooling/camel-debug-adapter#debug-adapter-for-apache-camel'
+
+    installer = util.OpenVsxInstaller (
+        type='camel',
+        repo='redhat/vscode-debug-adapter-apache-camel'
+    )
+
+    async def start(self, log: core.Logger, configuration: dap.ConfigurationExpanded):
+        install_path = util.vscode.install_path(self.type)
+        command = [
+            'java',
+            '-jar',
+            f'{install_path}/extension/jars/camel-dap-server.jar'
+        ]
+        return dap.StdioTransport(log, command)
+```
+
+# How to get Camel Debug Adapter server.jar file?
+- To get a jar file either you can compile the source or download from the Maven Central repository.
+- To download from Maven Central repository
+```sh
+cd /home
+curl -LO https://repo1.maven.org/maven2/com/github/camel-tooling/camel-dap-server/0.4.0/camel-dap-server-0.4.0.jar
```

Comment:
Would be nice to try with latest version released last 0.5.0 and if working to update with it